cms/static/js/features_jsx/studio/index.jsx

```diff
@@ -1,104 +1,131 @@
-import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import React from 'react';
 import ReactDOM from 'react-dom';
 
-class CourseOrLibraryListing extends React.Component {
+function CourseOrLibraryListing(props) {
+  /* global gettext */
+  /* eslint react/no-array-index-key: 0 */
```

Comment:
@arizzitano I added this because these lists are not dynamic. Therefore there is no risk of elements being added/removed in the wrong location in the list, and it is safe to just use the index as the key.